drivers/volume/portworx/pure.go

```diff
@@ -0,0 +1,20 @@
+package portworx
+
+const (
+	// PureDriverName is the name of the portworx-pure driver implementation
+	PureDriverName = "pure"
+)
+
+// pure is essentially the same as the portworx volume driver, just different in name. This way,
+// we can have separate specs for pure volumes vs. normal portworx ones
+type pure struct {
+	portworx
+}
+
+func (p *pure) Init(sched, nodeDriver, token, storageProvisioner, csiGenericDriverConfigMap string) error {
+	return p.portworx.init(sched, nodeDriver, token, storageProvisioner, csiGenericDriverConfigMap, PureDriverName)
+}
+
+func (p *pure) String() string {
```

Comment:
Let's also move the `ValidatePureVolumesNoReplicaSets` method from the portworx driver to this driver if possible. 
I would also recommend renaming it to `ValidateVolumeHasNoReplicaSets` and then let the portworx driver implement the DefaultDriver to use the NotImplemented version like other volume drivers. 

Does not have to be part of this PR.